src/content/docs/en/technology/overview/scroll-upgrades.mdx

```diff
@@ -17,6 +17,58 @@ The following contracts are used to initiate upgrades and execute upgrades after
 | L2 Scroll Multisig | Scroll| [`0xEfc9D1096fb65c832207E5e7F13C2D1102244dbe`](https://scrollscan.com/address/0xEfc9D1096fb65c832207E5e7F13C2D1102244dbe)|
 | L2 Timelock | Scroll | [`0xf6069DB81239E5194bb53f83aF564d282357bc99`](https://scrollscan.com/address/0xf6069DB81239E5194bb53f83aF564d282357bc99)|
 
+## Apr 2024 Bernoulli Upgrade
```

Comment:
Feb update didn't have a name, but should we lead with the name and include dates below? Esp since updates happen across months?

"Bernoulli Upgrade"

Reply:
ok! changed title of both updates